src/components/views/elements/AppTile.js

```diff
@@ -307,7 +308,6 @@ export default class AppTile extends React.Component {
                 if (this.iframe) {
                     // Reload iframe
                     this.iframe.src = this._sgWidget.embedUrl;
-                    this.setState({});
```

Comment:
Was this change related or fixing a different bug?

Reply:
This didn't do anything due to shallow merge, so no idea why it was even here. To force rerender (which didn't work btw)?

Reply:
Split it into a separate commit just in case.